### ISSUE TITLE
Allow Moments/Strategies to be bookmarked

### ISSUE THREAD

**Issue by LMulvey** [ifmeorg/ifme#1580]
**Bookmarks 🔖**

Users should have the ability to bookmark Moments and Strategies. This is identified by a column in the database for these items labeled `bookmark: boolean`

Users can click a bookmark toggle button on the Moment or Strategy they want to bookmark and it will add it to the bookmark list. Clicking the button again will remove the Moment or Strategy from the bookmark list.

[Sub-task for issue #187]

**Comment by dhuang612** [comment]
Hi,

Is this available to work on?

**Comment by GayathriVenkatesh** [comment]
Hi @LMulvey, is anyone presently working on this issue? If not, I'd love to take it up!

**Comment by LMulvey** [comment]
All yours, @GayathriVenkatesh :)
